src/core/graph.go

```diff
@@ -79,6 +79,14 @@ func (graph *BuildGraph) Target(label BuildLabel) *BuildTarget {
 func (graph *BuildGraph) TargetOrDie(label BuildLabel) *BuildTarget {
 	target := graph.Target(label)
 	if target == nil {
+		// TODO(jpoole): This is just a small usability message to help with the migration from v15 to v16. We should
```

Comment:
Wouldn't this still fire afterwards if you created one called `pleasings`? or am I missing something?

Reply:
This is only hit when:
1) the label they're looking for is within the pleasings sub-repo
2) we don't have any pleasings sub-repo configured
3) we're about to panic because of this

If we have a pleasings repo configured `_, ok := graph.subrepos["pleasings"]; !ok` would be false so we won't log this. 

I think this warning makes sense in that case?

Reply:
ah I see, yep, makes sense